plugins/ftdidmx/FtdiWidget-libftdi.cpp

```diff
@@ -84,74 +244,74 @@ bool FtdiWidget::Open() {
   }
 }
 
-bool FtdiWidget::Close() {
+bool FtdiInterface::Close() {
   if (ftdi_usb_close(&m_handle) < 0) {
-    OLA_WARN << Name() << " " << ftdi_get_error_string(&m_handle);
+    OLA_WARN << m_parent->Name() << " " << ftdi_get_error_string(&m_handle);
```

Comment:
It may be better to use m_parent->Description() here so that we have the serial number of the widget with the problem.